### ISSUE TITLE
Card browser: Increase Height of Column Headers

### ISSUE THREAD

**Issue by brishtibheja** [ankidroid/Anki-Android#17896]
In latest alpha, column headers' height are reduced which is a a11y issue. From David Allison:

> I /prefer/ it to be the height, so we get more information on the screen, but we're breaking Google accessibility guidelines on this [touchable elements should be 48dp tall]
> 
> Suggest that we hide it on scroll down, as is done with something like the statistics page header

**Comment by BrayanDSO** [comment]
Which columns in deck options are you refering to?

**Comment by brishtibheja** [comment]
Before:

![Image](https://github.com/user-attachments/assets/aaad5298-e88f-4b37-b522-ec2fb949695d)

Now:

![Image](https://github.com/user-attachments/assets/c0c383c7-94c5-4984-86da-41e8c42c8c21)

___

(does this make it clear?)

**Comment by BrayanDSO** [comment]
This is the browser, not deck options

**Comment by brishtibheja** [comment]
Ah, so sorry about that. often make these silly mistakes.

**Comment by argon2r** [comment]
![Image](https://github.com/user-attachments/assets/64789fbe-ef24-41ac-8a8c-93fe77720acf)
Is this the desired output? @brishtibheja

**Comment by brishtibheja** [comment]
@argon2r I am not a maintainer for this project; ask this in #android-design in the discord server if u want more feedback before opening a PR. personally, LGTM.
